[2025-03-13T12:45:00.609+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_etl.extract_weather_task scheduled__2025-03-13T12:40:00+00:00 [queued]>
[2025-03-13T12:45:00.616+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_etl.extract_weather_task scheduled__2025-03-13T12:40:00+00:00 [queued]>
[2025-03-13T12:45:00.617+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-13T12:45:00.630+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_weather_task> on 2025-03-13 12:40:00+00:00
[2025-03-13T12:45:00.636+0000] {standard_task_runner.py:60} INFO - Started process 248 to run task
[2025-03-13T12:45:00.639+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_data_etl', 'extract_weather_task', 'scheduled__2025-03-13T12:40:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/etl_repeat_weather.py', '--cfg-path', '/tmp/tmpk2ssmqkg']
[2025-03-13T12:45:00.641+0000] {standard_task_runner.py:88} INFO - Job 42: Subtask extract_weather_task
[2025-03-13T12:45:00.681+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_data_etl.extract_weather_task scheduled__2025-03-13T12:40:00+00:00 [running]> on host 7de7158dc909
[2025-03-13T12:45:00.897+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_etl' AIRFLOW_CTX_TASK_ID='extract_weather_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T12:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-13T12:40:00+00:00'
[2025-03-13T12:45:00.899+0000] {logging_mixin.py:188} INFO - Menggunakan data dummy karena API key tidak diatur.
[2025-03-13T12:45:00.917+0000] {python.py:201} INFO - Done. Returned value was: Data cuaca berhasil diambil pada 2025-03-13 12:45:00 untuk 5 kota
[2025-03-13T12:45:00.936+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_data_etl, task_id=extract_weather_task, execution_date=20250313T124000, start_date=20250313T124500, end_date=20250313T124500
[2025-03-13T12:45:00.971+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-13T12:45:00.992+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
